web/settings/modulconfiglp.php

```diff
@@ -1200,14 +1201,66 @@ function visibility_kia_advanced() {
 										<label for="soc_id_vin" class="col-md-4 col-form-label">VIN</label>
 										<div class="col">
 											<input class="form-control" type="text" name="soc_id_vin" id="soc_id_vin" value="<?php echo $soc_id_vinold ?>">
-											<span class="form-text small">
+											<spa-altn class="form-text small">
```

Comment:
Scheint ein Tippfehler zu sein?
```suggestion
											<span class="form-text small">
```